### PR TITLE
KEP-4193: bound service account token improvements

### PR DESCRIPTION
**Pull request by enj**
https://github.com/kubernetes/enhancements/issues/4193
https://github.com/kubernetes/enhancements/pull/4141
https://github.com/kubernetes/kubernetes/pull/120780

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -408,6 +418,39 @@ You can configure this behavior for the `spec` of a Pod using a
 [projected volume](/docs/concepts/storage/volumes/#projected) type called
 `ServiceAccountToken`.
 
+The token from this projected volume is a JSON Web Token (JWT).  The JSON payload of this
+token follows a well defined schema - an example payload for a pod bound token:
+
+```json
```
Comment (liggitt) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:424]: nit, mark this as `yaml` and use # for the comments

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 2]
```diff
@@ -408,6 +418,39 @@ You can configure this behavior for the `spec` of a Pod using a
...
+{
+  "aud": [
+    "https://kubernetes.default.svc"
```
Comment (liggitt) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:427]: this audience is also configurable, right? probably worth calling that out or people will hard-code stuff based on this doc

Comment (enj) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md:427]: Done.

**Pull request merged**